eZ/Publish/SPI/FieldType/Tests/FieldTypeTest.php

```diff
@@ -603,34 +602,19 @@ public function testAcceptGetEmptyValue()
 
     /**
      * @param mixed $inputValue
-     * @param \Exception $expectedException
      *
      * @dataProvider provideInvalidInputForAcceptValue
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
```

Comment:
If we check the occurrence of this exception should we add it in the docblock?

Reply:
We check for occurrence of dynamic `$exception` therefore the annotation is still required. Technically it's possible that the given `$exception` is gonna be different.